cmd/tracee-rules/main.go

```diff
@@ -104,7 +108,12 @@ func main() {
 			if err != nil {
 				return err
 			}
-			e, err := engine.NewEngine(sigs, inputs, output, os.Stderr, c.Bool("rego-enable-parsed-events"))
+
+			config := engine.Config{
+				ParsedEvents:        c.Bool("rego-enable-parsed-events"),
+				SignatureBufferSize: c.Uint(signatureBufferFlag),
```

Comment:
what if we pass -1? can we try to run with passing -1? the cli should output error.

Reply:
From a test right now:
`2022/02/03 11:44:10 invalid value "-1" for flag -sig-buffer: parse error` so it should be safe